packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/outputModels.ts

```diff
@@ -33,23 +33,23 @@ export interface ApplicationDataOutput {
   /** End date-time of the operation data, sample format: yyyy-MM-ddTHH:mm:ssZ. */
   operationEndDateTime?: string;
   /** Link for attachments. */
-  attachmentsLink?: string;
+  readonly attachmentsLink?: string;
   /** Optional boundary ID of the field for which operation was applied. */
```

Comment:
Do we have an example or testing to cover readOnly in input models?
My understanding is 

- If readonly is marked in output model, we'll tag as `readonly`
- If readonly is marked in input model, we'll remove these properties from input models

Reply:
This model is used in input models, https://github.com/qiaozha/autorest.typescript/blob/try-fix-modle-name-in-merge-and-patch/packages/autorest.typescript/test/smoke/generated/agrifood-data-plane/src/models.ts#L5-L54 we don't change anything which is the correct behavior.

Reply:
readonly has nothing to do with the http method. not sure why you mention PATCH.